*** node_modules/browser-sync/lib/server/index.js	2014-08-05 22:45:43.000000000 +0300
--- node_modules/browser-sync/lib/server/index.js	2014-09-10 12:58:44.720968838 +0300
***************
*** 149,155 ****
          }
      }

!     require("http").get(proxy.target, function (res) {
          res.on("data", function (data) {
              chunks.push(data);
          });
--- 149,155 ----
          }
      }

!     require(proxy.protocol === 'https' ? "https" : "http").get(proxy.target, function (res) {
          res.on("data", function (data) {
              chunks.push(data);
          });
