*** node_modules/browser-sync/node_modules/foxy/index.js	2014-07-18 18:09:14.000000000 +0300
--- node_modules/browser-sync/node_modules/foxy/index.js	2014-09-15 12:19:20.091373289 +0300
***************
*** 59,68 ****
  
      // Remove headers
      proxyServer.on("proxyRes", function (res) {
          if (res.statusCode === 302) {
              res.headers.location = utils.handleRedirect(res.headers.location, opts, proxyHost);
          }
-         utils.removeHeaders(res.headers, ["content-length", "content-encoding"]);
      });
  
      function getRules() {
--- 59,70 ----
  
      // Remove headers
      proxyServer.on("proxyRes", function (res) {
+         if (res.statusCode === 301) {
+             res.statusCode = 302;
+         }
          if (res.statusCode === 302) {
              res.headers.location = utils.handleRedirect(res.headers.location, opts, proxyHost);
          }
      });
  
      function getRules() {
